content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -24,7 +24,7 @@ API or the `kube-*` components from the upstream code, see the following instruc
 - You need to have these tools installed:
 
   - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-  - [Golang](https://golang.org/doc/install) version 1.13+
+  - [Golang](https://go.dev/doc/install) version 1.13+
```

Comment:
Golang vervsion is determined by the kubernetes/kubernetes repository.

Reply:
Should i remove v1.13+ from here @tengqm ?

Reply:
This version is a moving target. I'd suggest we use v1.20+ for golang as of today.